Task2/Authentication/ApplicationUser.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace Task2.Authentication
+{
+    public class ApplicationUser : IdentityUser
+    {
+    }
```

Comment:
Why this model is empty? Can we customize this?

Reply:
I inherited Application user from IdentityUser , to get all its properties and for now Application user is empty but I can add other properties for user too here.just to keep this option open its inherited from IdentityUser.